src/components/NftRewards/NftPreview.tsx

```diff
@@ -29,6 +34,26 @@ export function NftPreview({
       rewardTier.maxSupply !== DEFAULT_NFT_MAX_SUPPLY,
   )
 
+  const contentType = useContentType(fileUrl)
+  const isVideo = contentType === 'video/mp4'
+  const isLoading = contentType === undefined
```

Comment:
**suggestion** we should update `useContentType` to have a `loading` flag instead of checking for undefined.

Reply:
```js
  const { contentType, loading } = useContentType(fileUrl);
```